app/src/main/java/in/ac/bits_pilani/goa/ard/utils/AHC.java

```diff
@@ -58,4 +58,61 @@
      * background image for navigation drawer.
      */
     public static final String FDR_NAV_DRAWER_IMAGE_LIST = "navDrawerImages";
+
+    /**
+     * Firebase Database Reference for aboutMAC.
+     */
+    public static final String FDR_ABOUT_MAC = "aboutMAC";
+
+    /**
+     * String shtml.
+     */
+    public static final String TEXT_ABOUT_MAC = "text";
+
+    /**
+     * String surl.
+     */
+    public static final String IMAGEURL_ABOUT_MAC = "imageUrl";
+
+    /**
+     * Float magicnumber.
```

Comment:
please mention in the comments, what this magic number is used for...
and avoid naming it so if possible